notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -626,36 +626,52 @@
   },
   {
    "cell_type": "markdown",
-   "id": "f3490c8a-5ee8-44cd-ae5e-26a6ca7b4017",
+   "id": "1497c44e-7b82-4646-b00f-e910688bfb3d",
    "metadata": {},
    "source": [
-    "### Install docling-sdg\n",
+    "### Select the chunks for the seed examples\n",
+    "\n",
+    "Chunks for seed examples should be diverse in style. These can be selected by hand or selecting diverse chunks from all of the chunks using the [subset selection notebook](https://github.com/instructlab/examples/blob/main/notebooks/instructlab-knowledge/subset-selection.ipynb).\n",
     "\n",
-    "[Docling-sdg](https://github.com/docling-project/docling-sdg) project is used to generate question and answer pairs for seed examples."
+    "If users are selecting chunks by hand, chunks should be taken directly from lines in `chunks.jsonl`. These lines have `chunk`, `file`, and `metadata` fields for each entry.\n",
+    "\n",
+    "The below code randomly selects a preset number of chunks and saves them in a jsonl file for the next step."
```

Comment:
Might be a good idea to save this to an intermediate file - `chunks-candidates.jsonl` maybe - so that it's impossible for a user to have hand curated a `chunks.jsonl` file and then overwrite it by running this cell.